source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -778,6 +778,9 @@ def __init__(self):
     def fill_journeys(self, request_type, api_request, instance):
         logger = logging.getLogger(__name__)
 
+        if api_request['max_nb_journeys'] is not None and api_request['max_nb_journeys'] <= 0:
```

Comment:
if `max_nb_journeys` if of type `inputs.positive`, you might not need to check if it's below zero.

Reply:
it's a safe check, just in case someone overwrite it like this:
https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/interfaces/v1/Journeys.py#L532